Pattern '/[abc]+' matches /builds which is where gitlab clones the repo into.
diff --git a/cmd/gdu/app/app_test.go b/cmd/gdu/app/app_test.go
index aa74337..56f677e 100644
--- a/cmd/gdu/app/app_test.go
+++ b/cmd/gdu/app/app_test.go
@@ -114,7 +114,7 @@ func TestAnalyzePathWithIgnoring(t *testing.T) {
 	out, err := runApp(
 		&Flags{
 			LogFile:           "/dev/null",
-			IgnoreDirPatterns: []string{"/[abc]+"},
+			IgnoreDirPatterns: []string{"/[xyz]+"},
 			NoHidden:          true,
 		},
 		[]string{"test_dir"},
